src/main/java/com/amazon/aocagent/tasks/ECSClean.java

```diff
@@ -0,0 +1,68 @@
+package com.amazon.aocagent.tasks;
+
+import com.amazon.aocagent.enums.GenericConstants;
+import com.amazon.aocagent.models.Context;
+import com.amazon.aocagent.services.EC2Service;
+import com.amazon.aocagent.services.ECSService;
+import com.amazonaws.services.ec2.model.Instance;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+@Log4j2
+public class ECSClean implements ITask {
+  private EC2Service ec2Service;
+  private ECSService ecsService;
+
+  @Override
+  public void init(Context context) throws Exception {
+    this.ec2Service = new EC2Service(context.getStack().getTestingRegion());
+    this.ecsService = new ECSService(context.getStack().getTestingRegion());
+  }
+
+  @Override
+  public void execute() throws Exception {
+    cleanTasks();
+    cleanContainerInstances();
+    cleanCluster();
+    cleanTaskDefinitions();
+  }
+
+  private void cleanTaskDefinitions() {
+    ecsService.cleanTaskDefinitions(GenericConstants.AOC_PREFIX.getVal());
+  }
+
+  private void cleanCluster() {
```

Comment:
how do we clean the clusters which are older than 2 hours?

Reply:
I can add it into cleanCLuster method.  every cluster name suffix with timestamp when it created